app/services/foreman_bootdisk/iso_generator.rb

```diff
@@ -217,25 +217,22 @@ def self.fetch(path, uri, limit = 10)
           else
             http_object = Net::HTTP
           end
-          http_object.start(uri.host, uri.port) do |http|
-            request = Net::HTTP::Get.new(uri.request_uri, 'Accept-Encoding' => 'plain')
-
-            http.request(request) do |response|
-              case response
-              when Net::HTTPSuccess then
-                response.read_body do |chunk|
-                  file.write chunk
-                end
-              when Net::HTTPRedirection then
-                raise("Too many HTTP redirects when downloading #{uri}") if limit <= 0
-
-                fetch(path, response['location'], limit - 1)
-                # prevent multiple writes to the cache
-                write_cache = false
-              else
-                raise ::Foreman::Exception, N_(format("Unable to download boot file %{uri}, HTTP return code %{code}", uri: uri, code: response.code))
-              end
+
+          response = http_object.get_response(uri)
```

Comment:
the `Accept-Encoding` was added for this reason https://projects.theforeman.org/issues/25992, does it now work even without?